docs/user-guide/aws-examples/eks.rst

```diff
@@ -0,0 +1,215 @@
+   apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: nemo-workspace
+   spec:
+     accessModes:
+       - ReadWriteMany
+     storageClassName: efs-sc
+     resources:
+       requests:
+         storage: 150Gi
+   EOF
+
+This PVC requests 150GiB of storage with ReadWriteMany access mode from the efs-sc StorageClass.
+
+Dask cluster creation:
+----------------------
+
+Please refer to kubernetescurator.rst for instructions on creating a Docker secret to utilize the NeMo image and upload data to the PVC created in the previous step.
+
+Python environment can be setup by executing the following commands:
+
+.. code-block:: bash
+
+    python3 -m venv venv
+    source venv/bin/activate
+
+    pip install dask_kubernetes
+    pip install kubernetes
+
+The environment to run the provided scripts needs only dask-kubernetes and kubernetes packages.
+
+.. code-block:: bash
+
+  python3 examples/k8s/create_dask_cluster.py \
+          --name dask-gpu-cluster \
+          --n_workers 2 \
+          --image nvcr.io/nvidian/nemo:nightly \
+          --image_pull_secret ngc-registry \
+          --pvcs nemo-workspace:/nemo-workspace
+
+The above command uses the create_dask_cluster python code to create 2 GPU dask workers with PVCs attached to the dask-gpu-cluster.
+
+After the cluster is created, you can check if the scheduler and worker pods are running by executing:
+
+.. code-block:: bash
+
+    kubectl get pods
+
+The output will look as follows:
+
++---------------------------------------------------------+-------+---------+----------+------+
+| NAME                                                    | READY | STATUS  | RESTARTS | AGE  |
++---------------------------------------------------------+-------+---------+----------+------+
+| dask-kubernetes-operator-1720671237-6f8c579d4d-gk8pg    | 1/1   | Running | 0        | 27h  |
++---------------------------------------------------------+-------+---------+----------+------+
+| rapids-dask-default-worker-be7c9e6b19-668b8cc459-cxcwg  | 1/1   | Running | 0        | 21h  |
++---------------------------------------------------------+-------+---------+----------+------+
+| rapids-dask-default-worker-f4b5c0ff1a-66db8c4cb5-w68gd  | 1/1   | Running | 0        | 21h  |
++---------------------------------------------------------+-------+---------+----------+------+
+| rapids-dask-scheduler-5dfc446f-9tw2t                    | 1/1   | Running | 0        | 21h  |
++---------------------------------------------------------+-------+---------+----------+------+
+
+
+
+Use Kubernetes Python client library to submit NeMo-Curator jobs to the Dask scheduler:
+
+------------------------------------------------------
+
+In this method, we programmatically connect to the scheduler pod using the Kubernetes Python client library to execute the existing NeMo curator modules.
+
+This approach can be used when employing another wrapper or service to submit jobs to Dask cluster in a distributed manner.
```

Comment:
I think you can delete this line.

> submit jobs

ALso, I just want to clarify the purpose of Dask real quick since I see this phrasing of "jobs" and "submitting jobs" a few times in the docs. Dask is not like SLURM or another job scheduler. It is a framework that automatically handles scaling Python code across multiple processes or nodes. Here is a simple example of Python code that uses Dask:
```python
import dask.dataframe as dd

# Create a Dask DataFrame
df = dd.read_csv('large_dataset.csv')

# Perform operations (these are executed in parallel)
result = df.groupby('column').mean().compute()
```
All of the computation you see will be executed on the Dask cluster without the need for explicit job submission or anything else. You should think of Dask as being more like PyTorch than SLURM. I'd recommend looking at the [10-minutes to Dask](https://docs.dask.org/en/stable/10-minutes-to-dask.html) page for more details.